diff -up a/data/org.gnome.frogr.appdata.xml.in b/data/org.gnome.frogr.appdata.xml.in
--- a/data/org.gnome.frogr.appdata.xml.in.fix	2017-09-26 17:08:41.330724727 +0200
+++ b/data/org.gnome.frogr.appdata.xml.in	2017-09-26 17:08:47.264644794 +0200
@@ -2,7 +2,7 @@
 <component type="desktop">
   <id>frogr.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <name>frogr</name>
+  <name>org.gnome.frogr</name>
   <summary>frogr: a Flickr Remote Organizer for GNOME</summary>
   <description>
     <p>
